README.md

```diff
@@ -52,12 +52,12 @@ teosd --datadir=<path_to_dir>
 
 ### Running `teosd` in another network
 
-By default, `teosd` runs on `mainnet`. In order to run it on another network, you need to change the network parameter in the configuration file or pass the network parameter as a command-line option. Notice that if `teosd` does not find a `bitcoind` node running in the same network that it is set to run, it will refuse to run.
+By default, `teosd` runs on mainnet (`bitcoin`). In order to run it on another network, you need to change the network parameter in the configuration file or pass the network parameter as a command-line option. Notice that if `teosd` does not find a `bitcoind` node running in the same network that it is set to run, it will refuse to run.
 
 The configuration file option to change the network where `teosd` will run is `btc_network`:
 
 ```
-btc_network = mainnet
+btc_network = bitcoin
```

Comment:
We're going to have an issue here, rust bitcoin reports this as `bitcoin` while `bitcoind` reports this as main. Not sure what `bitcore-rpc` does.

Reply:
This might be confusing. But the input teosd takes accepts `bitcoin` and not `mainnet`.

Reply:
Yeah, I think I took that from `CoreLN`. What does  [GetBlockchainInfoResult](https://docs.rs/bitcoincore-rpc-json/0.16.0/bitcoincore_rpc_json/struct.GetBlockchainInfoResult.html#) reports `chain` as for mainnet?

We should either stick to that, or convert it while parsing the config values

Reply:
`Configuration error: btc_network conflict. bitcoind is running on main but teosd is set to run on bitcoin` it actually reports main :facepalm:

Reply:
That makes sense actually, given that's parsed directly from `bitcoind` which reports `main` 😅

Reply:
We can accept both really, but cast it to `main` if required since that's what it will be checked against

Reply:
Fine. FYI, bitcoin chain returns these:
main -> main
testnet -> test
signet -> signet
regtest -> regtest

I will make that we accept either main/mainnet & test/testnet.

Reply:
Makes sense.